package.json

```diff
@@ -5,6 +5,7 @@
   "main": "./lib/distributed-callback-queue.js",
   "scripts": {
     "test": "yarn lint && mdep test run",
+    "test:local": "yarn lint && yarn compile && jest __tests__/*.js",
```

Comment:
this seemed pretty useful for iterative testing without a Redis container, but that's my opinion

i am more than willing to omit this change if you prefer

Reply:
you can do `yarn test -C`, which would not destroy containers, however, I don't see any problems with an extra helper that does some manual testing